tests/integration/hubapp-dind/test_integration.sh

```diff
@@ -0,0 +1,30 @@
+set -e
+
+if [ "${PWD##*/}" != "jina" ]
+  then
+    echo "test_integration.sh should only be run from the jina base directory"
+    exit 1
+fi
+
+CONTAINER_ID=$(docker run -v /var/run/docker.sock:/var/run/docker.sock --network=host -d jinaai/test_hubapp_dind)
```

Comment:
As we can see here, in order for `hubapp` to be able to connect to `hubpod` they need to be in the same network, meaning that `hubapp` needs to be started with `--network==host`

Reply:
By setting `--network==host` for `container#1`, the `container#1` will share all its port with the physical machine. As for the alternative solution you mentioned above, do you mean `container#2` also need `--network==host`?

```
hubapp (container#1)
    |- gateway
    |- pod1
    |    |- pea1_1
    |        |- hubpod (container#2)
    |- ...
```

Reply:
I do not understand which alternative solution you refer to?

Reply:
But `container #2` is the one spawned by `ContainerPea` and for now we always start it with `--network=host` (because we expect to be available from the outside). I think this is a good thing to have. (I do not think we should do (or even if it is possible) ) to do the "reflection" that this `pea1_1` is also living inside a `Container` and therefore `pea1_1` should be attached to the `bridge` network where I live and with a specific IP that I know.